### PR TITLE
Bromley waste phase2 dd processing updates

### PR DESCRIPTION
**Pull request by struan**
[skip changelog]

### CODE REVIEW

File: perllib/Integrations/Pay360.pm [hunk 1]
```diff
@@ -202,6 +202,37 @@ sub get_recent_payments {
...
+        if ($res->{StatusCode} eq 'SA') {
+            if ($res->{CancelledPayerRecords}) {
+                return force_arrayref( $res, 'CancelledPayerRecords' );
+            } else {
+                return [];
+            }
+        } else {
+            return {
+                error => $res->{StatusMessage}
+            }
```
Comment (davea) [inline perllib/Integrations/Pay360.pm:229]: Missing semicolon here.

File: perllib/FixMyStreet/Cobrand/Bromley.pm [hunk 2]
```diff
@@ -1251,10 +1251,10 @@ sub waste_payment_type {
     my ($self, $type, $ref) = @_;
 
     my ($sub_type, $category);
-    if ( $type eq 'Payment: 01' ) {
+    if ( $type eq 'Payment: 01' || $type eq 'First Time' ) {
         $category = 'Garden Subscription';
         $sub_type = $self->waste_subscription_types->{New};
-    } elsif ( $type eq 'Payment: 17' ) {
+    } elsif ( $type eq 'Payment: 17' || $type eq 'Regular' ) {
```
Comment (davea) [inline perllib/FixMyStreet/Cobrand/Bromley.pm:1261]: Should the tests continue to include `Payment: 17` and `Payment: 01` too?

**Review by davea — CHANGES_REQUESTED** [review]
Couple of small questions is all.

File: perllib/FixMyStreet/Cobrand/Bromley.pm (hunk 2 above)
Comment (struan) [reply perllib/FixMyStreet/Cobrand/Bromley.pm:1261]: I'm not sure there ever used in practice but given the docs and sample data disagree it seemed best to hedge, so yes.

**Review by davea — APPROVED** [review]
👍

**Pull request closed without merging**